GNUmakefile

```diff
@@ -11,7 +11,7 @@ PROVIDER_TYPE=rediscloud
 .PHONY: testacc install_macos website website-test
 
 testacc:
-	TF_ACC=1 go test ./... -v $(TESTARGS) -timeout 120m
+	TF_ACC=1 go test ./... -v $(TESTARGS) -timeout 120m -parallel=1
```

Comment:
I'd probably just change the tests from `resource.ParallelTest` to `resource.Test` rather than just defining the parallel limit

Reply:
I think we will still try later to have parallel runs, that's why it's configurable at the test level.